contxt/services/contxt_deployments.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Optional
+
+from requests.exceptions import HTTPError
+
+from ..auth import Auth
+from ..models.contxt import Cluster
+from .api import ApiEnvironment, ConfiguredApi
+
+
+class ContxtDeploymentService(ConfiguredApi):
+    """Contxt API client"""
+
+    _envs = (
+        ApiEnvironment(
+            name="production",
+            base_url="https://contxt.api.ndustrial.io/deploy/v1",
+            client_id="8qY2xJob1JAxhmVhIDLCNnGriTM9bct8",
+        ),
+        ApiEnvironment(
+            name="staging",
+            base_url="https://contxt-api.staging.ndustrial.io/deploy/v1",
+            client_id="qGzdTXcmB57zlTp86rYsivG9qEss1lbF",
+        ),
+    )
+
+    def __init__(self, env: str, auth: Auth, **kwargs) -> None:
```

Comment:
to be consistent:
```suggestion
    def __init__(self, auth: Auth, env: str = "production", **kwargs) -> None:
```

the reasoning behind defaulting this is because `production` is by far common case